streams/src/test/java/org/apache/kafka/streams/state/internals/InMemoryKeyValueLoggedStoreTest.java

```diff
@@ -39,18 +40,16 @@
             Class<V> valueClass,
             boolean useContextSerdes) {
 
-        StateStoreSupplier supplier;
-        if (useContextSerdes) {
-            supplier = Stores.create("my-store").withKeys(context.keySerde()).withValues(context.valueSerde())
-                .inMemory().enableLogging(Collections.singletonMap("retention.ms", "1000")).build();
-        } else {
-            supplier = Stores.create("my-store").withKeys(keyClass).withValues(valueClass)
-                .inMemory().enableLogging(Collections.singletonMap("retention.ms", "1000")).build();
-        }
+        StoreBuilder storeBuilder = Stores.keyValueStoreBuilder(
```

Comment:
If `useContextSerdes` is false, we should use `Serdes.serdeFrom(keyClass)` and `Serdes.serdeFrom(valueClass)` since the context may not set the serdes.

Reply:
EDIT: actually, I think it does not matter since the context serdes are always the same as the passed in class types. So we can just remove this in a follow-up tech debt cleanup PR.